### ISSUE TITLE
IED does nothing

### ISSUE THREAD

**Issue by Spacedong** [HippieStationCode/HippieStation13#3128]
no like I'm not joking or over-exaggerating here it literally creates a negative-value explosion that cannot do anything

**Comment by yobob591** [comment]
WEW

**Comment by Pyko1** [comment]
```
	if(det_time < 0) //checking for 'duds'
		range = 1
		det_time = rand(30,80)
	else
		range = pick(2,2,2,3,3,3,4)
```
`explosion(src.loc,-1,-1,-1, flame_range = range)	// no explosive damage, only a large fireball.`

Should work, it is supposed to only make the flame effect as far as i know.

**Comment by banthebantz** [comment]
I'm pretty sure on other servers it makes a tiny explosion at least, enough to knock someone out and cause some damage. Should be buffed tbh.

**Comment by Pyko1** [comment]
TG has buffed it to 
`	explosion(src.loc,-1,-1,2, flame_range = 4)`
since our last rebase, flame range is silly tho, they have the code for random size but then just use 4.

But do remember that any explosion is quite likely to blow off bodyparts around here.

**Comment by Spacedong** [comment]
it doesnt blow off bodyparts its not strong enough, its just a fireball

**Comment by Spacedong** [comment]
I want it to do what banthebantz said

**Comment by Pyko1** [comment]
yes, but it will start blowing of hands and legs if any other than flame_range is buffed.

**Comment by GuyonBroadway** [comment]
That's what a single tank bomb does Pyko and can be pretty devastating is used right.

**Comment by GuyonBroadway** [comment]
What if IED construction also required a few metal rods for "shrapnel" and released a burst of them upon detonation.

**Comment by Pyko1** [comment]
At least currently IEDs are piss easy to massproduce so i doubt it would be good idea to add explosion without adding something rarer to recipe.

**Comment by Swagile** [comment]
you used IEDs to blow up fueltank bombs remotely, but thats all their good for right now, because a flamethrower can be made just as easily and plasma tank stolen for 100x the effect that IED's currently have

make it produce some plasma when it explodes and rename it to "firebomb" and now it actually has a use besides "lol shit tier fire on a random timer" or "fuel tank primer"

**Comment by Pyko1** [comment]
What if it could actually use timers and shit, and only without one its random time. along with spawning plasma puff.

**Comment by ArcLumin** [comment]
It's literally just used to do damage. It can gib, but it'll only gib corpses if you place it directly on their corpse.

**Comment by Swagile** [comment]
it can gib?????

holy shit, i thought the shitty explosion could not do that

now i know what to do during ling rounds if i can't get crematorium / gibber access

**Comment by Spacedong** [comment]
it cant gib

**Comment by GeneralGoose** [comment]
it's called improvised EXPLOSIVE device why the fuck is it incendiary
